var/spack/repos/builtin/packages/aocl-sparse/package.py

```diff
@@ -108,7 +108,7 @@ def cmake_args(self):
         if spec.satisfies("@3.0:"):
             args.append(self.define_from_variant("BUILD_ILP64", "ilp64"))
 
-        if self.spec.satisfies("@4.0:"):
+        if self.spec.satisfies("@4.1:"):
```

Comment:
Can you give some context on this change?

Reply:
This is a fix for https://github.com/spack/spack/issues/43193

Reply:
New PR raised - sparse fix for issue https://github.com/spack/spack/issues/43193